network_path:
  value: rsc/jsh/ver3
  dtype: string

model_number:
  value: 37400
  dtype: string

box_geometry:
  value: [0.3, 0.6, 0.45]
  dtype: vector

plugin:
  raisin_iekf_plugin: # plugin package name
    rate:
      value: 500 # loop frequency
      dtype: double
      
#  raisin_vicon_plugin: # plugin package name
#    rate:
#      value: 500 # loop frequency
#      dtype: double
